koku/koku/dev_middleware.py

```diff
@@ -67,7 +67,7 @@ def process_request(self, request):  # pylint: disable=no-self-use
             }
 
         """
-        if hasattr(request, "META") and settings.DEVELOPMENT_IDENTITY:
+        if hasattr(request, "META") and hasattr(settings, "DEVELOPMENT_IDENTITY"):
             identity_header = settings.DEVELOPMENT_IDENTITY
```

Comment:
This change allows for a falsey DEVELOPMENT_IDENTITY which the former code was apparently checking. Does this case still need to be handled?

Reply:
Fixed! Nice catch! ^_^